src/app/hooks/useScreensize.ts

```diff
@@ -0,0 +1,11 @@
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { useTheme } from '@mui/material/styles'
+import { Theme } from '@mui/material/styles/createTheme'
+
+export const useScreenSizeOnTheme = (theme: Theme) => ({
+  isMobile: useMediaQuery(theme.breakpoints.down('sm')),
+  isTablet: useMediaQuery(useTheme().breakpoints.down('md')),
```

Comment:
There is no consistency, `isMobile` relies on function arg and `itTablet`/`isDesktop` use MUI hook directly.

Reply:
Ah sorry, fixed.